README.md

```diff
@@ -15,24 +17,45 @@ Utils folder is just for general functions you want to use in several places.
 And finally in the root folder (src) I only place the index.js file, the serviceWorker.js and the tests.
 
 # How to install it?
+
 Just run this code:
-``` bash
+
+```bash
 git clone https://github.com/NauCode/ReactJS-Project-Structure.git your_project_name
```

Comment:
Change this to our front end repository url [ https://github.com/BuildForSDGCohort2/Team-tech-talents-Frontend.git ]